_includes/head.html

```diff
@@ -4,7 +4,14 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
 
     <title>{% if page.title %}{{ page.title }}{% else %}{{ site.title }}{% endif %}{% if page.tagline %} || {{page.tagline}}{%endif%} || OpenTechSchool.org</title>
-    <meta name="description" content="{{ site.description }}">
+    <meta name="description" content="{{ site.description }}" />
+	  <meta name="author" content="OpenTechSchool">
```

Comment:
With the document type we are using (`<!DOCTYPE html>`) the self-closing slash is not needed.